Description:
 Disable JavaRebelIntegration (no yet in Debian and non-free stuff)
Forwarded: no
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2009-09-27
--- a/build.xml
+++ b/build.xml
@@ -207,6 +207,7 @@
       bootclasspath="${boot.classpath.j2se1.4}"
       excludes="
         freemarker/ext/beans/_EnumModels.java,
+        freemarker/ext/beans/JRebelClassChangeNotifier.java,
         freemarker/ext/dom/SunInternalXalanXPathSupport.java,
         
         freemarker/ext/jsp/**,
